### DISCUSSION TITLE
Just two ideas

### DISCUSSION THREAD

**Discussion by miaochiahao** [projectdiscovery/nuclei-templates#702] [Ideas]
Here are two ideas:
* An editor online to convert burp request to nuclei-template yaml automatically
* A tool that parses [jaeles-signatures](https://github.com/jaeles-project/jaeles-signatures/blob/master/cves/apache-ofbiz-xss-cve-2020-9496.yaml) (something like nuclei-template) into nuclei-template

Is anyone working on those ideas? If not, I will work on that.

**Comment by ehsandeep** [comment]
@miaochiahao we will do the editor part in the coming days.
